core/controllers/diagnostic_test_player_test.py

```diff
@@ -28,53 +25,6 @@
 from core.tests import test_utils
 
 
-class DiagnosticTestLandingPageTest(test_utils.GenericTestBase):
-    """Test class for the diagnostic test player page."""
-
-    def setUp(self) -> None:
-        super().setUp()
-        self.signup(self.CURRICULUM_ADMIN_EMAIL, self.CURRICULUM_ADMIN_USERNAME)
-        self.owner_id = self.get_user_id_from_email(self.CURRICULUM_ADMIN_EMAIL)
-        self.original_parameter_registry = (
-            platform_parameter_registry.Registry.parameter_registry.copy())
-
-    def tearDown(self) -> None:
-        super().tearDown()
-        platform_parameter_registry.Registry.parameter_registry = (
-            self.original_parameter_registry)
-
-    def test_should_not_access_diagnostic_test_page_when_feature_is_disabled(
-        self) -> None:
-        feature_flag_services.update_feature_flag(
-            feature_flag_list.FeatureNames.DIAGNOSTIC_TEST.value,
-            False,
-            0,
-            []
-        )
-        self.get_html_response(
-            feconf.DIAGNOSTIC_TEST_PLAYER_PAGE_URL,
-            expected_status_int=404
-        )
-
-    def test_should_access_diagnostic_test_page_when_feature_is_enabled(
-        self) -> None:
-        self.get_html_response(
-            feconf.DIAGNOSTIC_TEST_PLAYER_PAGE_URL,
-            expected_status_int=404
-        )
-
-        feature_flag_services.update_feature_flag(
-            feature_flag_list.FeatureNames.DIAGNOSTIC_TEST.value,
-            True,
-            0,
-            []
-        )
-        self.get_html_response(
-            feconf.DIAGNOSTIC_TEST_PLAYER_PAGE_URL,
-            expected_status_int=200
-        )
-
```

Comment:
Maybe move these tests to access_validator test*

Reply:
Done